### ISSUE TITLE
[Android] CrossMediaManager.Current.StatusChanged occurs 2 times if user click button to Pause the audio

### ISSUE THREAD

**Issue by jerryno6** [martijn00/XamarinMediaManager#303]
1. I Play 1 file audio
await CrossMediaManager.Current.Play(fullFilePath);

**2. I clicked on button Pause to call method
CrossMediaManager.Current.PlaybackController.PlayPause()
I see that
CrossMediaManager.Current.StatusChanged occured 2 times, both have status: "Paused"**

after that, If I click button Play to continue playing the audio, 
CrossMediaManager.Current.StatusChanged occured 1 time with status "Playing", this time is ok

**Comment by jerryno6** [comment]
Sorry, because of my codes.

**Comment by mike-rowley** [comment]
What platform and which test project can you reproduce this with?

**Comment by jerryno6** [comment]
I test it on Android 7.1.
I will write a simple app to reproduce this later.

**Comment by mike-rowley** [comment]
I have tested it using the Sample Android App and cannot reproduce this behavior.  I would recommend comparing what you are doing to what is in the Android Sample App to see if you can see what the issue may be.

**Comment by jerryno6** [comment]
I've just tested on the SampleProject (Sample/MediaForms.Android) of Plugin MediaManager.

I change this
        ```
private async void StopButton_OnClicked(object sender, EventArgs e)
        {
           //await CrossMediaManager.Current.PlaybackController.Stop();
            **await CrossMediaManager.Current.PlaybackController.PlayPause();**
        }
```
+at First, I touch button **Play Audio (show ID3 Metadata)**
+Each time I touch button Stop, to change status to Pause, method CurrentOnStatusChanged occurs two times, with status=Pause. But when i Click Stop again to change status to Playing, method CurrentOnStatusChanged occurs one time with status = Playing

**Comment by mike-rowley** [comment]
This issue is fixed by PR#307
